snuba/manual_jobs/scrub_users_from_eap_spans_str_attrs.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+from typing import Any, Mapping, Optional
+
+from snuba.clusters.cluster import ClickhouseClientSettings, get_cluster
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.manual_jobs import Job, JobLogger, JobSpec
+
+_SCRUBBED = "scrubbed"
+
+
+class ScrubUserFromEAPSpansStrAttrs(Job):
+    def __init__(self, job_spec: JobSpec) -> None:
+        self.__validate_job_params(job_spec.params)
+        super().__init__(job_spec)
+
+    def __validate_job_params(self, params: Optional[Mapping[Any, Any]]) -> None:
+        assert params
+        assert isinstance(params["organization_ids"], list)
+        assert all([isinstance(p, int) for p in params["organization_ids"]])
+        self._organization_ids = params["organization_ids"]
+        self._start_datetime = datetime.fromisoformat(params["start_datetime"])
+        self._end_datetime = datetime.fromisoformat(params["end_datetime"])
+
+    def _get_query(self, cluster_name: str | None) -> str:
+        organization_ids = ",".join([str(p) for p in self._organization_ids])
+        start_datetime = self._start_datetime.isoformat()
+        end_datetime = self._end_datetime.isoformat()
+        on_cluster = f"ON CLUSTER '{cluster_name}'" if cluster_name else ""
+        return f"""ALTER TABLE spans_str_attrs_3_local
+{on_cluster}
+DELETE WHERE (attr_key = 'sentry.user.ip' OR attr_key = 'sentry.user')
```

Comment:
I think we are scrubbing right? 'scrubbed' for user.ip and 'ip:scrubbed` for user

Reply:
you can delete too, it's file

Reply:
it wont let me `alter table update` bc the columns are in the primary key or something, i have to do a delete